exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -328,9 +432,11 @@
       //Iterates through all the nodes for update
       function reloadMetrics() {
           for (i = 1; i <= size; i++) {
-              var address = $("#row-"+i).find("#address").contents().get(0).nodeValue.trim();
-              var httpPort = $("#row-"+i).find("#httpPort").contents().get(0).nodeValue.trim();
-              updateMetricsHtml(address, httpPort, i);
+              if ( $("#row-"+i).find("#stateKey").length == 0 ) {
```

Comment:
This check means we are not loading the metrics for newly available Drillbits, why is that ? 

I think instead we should check if the `status is ONLINE` then only `updateMetricsHtml`.

Reply:
Very odd. The `stateKey` seems to disappear (or never appeared). Let me look at what happened.